env-macos.sh

```diff
@@ -4,7 +4,9 @@
 # It sets the LLVM environment variables.
 
 MLIR_SYS_170_PREFIX="$(brew --prefix llvm@17)"
+LLVM_SYS_170_PREFIX="$(brew --prefix llvm@17)"
 TABLEGEN_170_PREFIX="$(brew --prefix llvm@17)"
 
 export MLIR_SYS_170_PREFIX
 export TABLEGEN_170_PREFIX
+export LLVM_SYS_170_PREFIX
```

Comment:
Please keep the final newline.

If you're using code or codium you may insert this into your configuration to insert them automatically (and two extra lints):
```
  "files.trimFinalNewlines": true,
  "files.trimTrailingWhitespace": true,
  "files.insertFinalNewline": true,
```

Reply:
Fixed! Thank you